### ISSUE TITLE
Add card view to images 

### ISSUE THREAD

**Issue by fzyang1227** [Code-4-Community/speak-for-the-trees-frontend#316]
**Blocked by**
#292 

**Why**
- ability to view uploaded images to display on the approval page
- see #293 

**What to do**
- implement a grid view of the unapproved images to be reviewed
- create a component for each individual grid portion
- refer to [figma design](https://www.figma.com/file/kDLySmyjpw2dorjTkuTuBB/SFTT-2023-2024?type=design&node-id=1196-4068&mode=design&t=kbLCz3L7kGNSDVJu-4)
    - _notes: can ignore the recently rejected section_
    - filters include site_id and date submitted (no need for the status anymore, also for the list view no need for status)
    - the following two views by getting all the relevant info for the images that are unapproved (see: https://github.com/Code-4-Community/speak-for-the-trees-backend-v2/issues/202)

*specific features to note:*
1. the top with column names to sort by that feature
2. filters to filter the unapproved images
3. _(in grid view)_ hover for the species name, click to be added to eventually be approved

**popups for expanding on the image will be implemented in another ticket**

![Image](https://github.com/Code-4-Community/speak-for-the-trees-frontend/assets/62119227/a54e40de-74b4-4b26-ad64-cc54b6412b26)
